www/front_src/src/Resources/index.tsx

```diff
@@ -129,14 +137,30 @@ const Resources = (): JSX.Element => {
       .finally(() => setLoading(false));
   };
 
+  const initAutorefresh = (): void => {
+    window.clearInterval(refreshIntervalRef.current);
```

Comment:
```suggestion
    clearInterval(refreshIntervalRef.current);
```

Reply:
typescript does not work with this change

Reply:
https://github.com/microsoft/TypeScript/issues/842#issuecomment-339860768